README.md

```diff
@@ -365,33 +390,27 @@ negate([
 
 ## 2.9.9 Z1 Trace Operator
 
-Returns the previous input [trace](#23-trace).
-
-TODO: Decide whether the feedback operators belong alongside regular operators, or if they should be considered special, because they tend to be used in special feedback nodes that introduce cycles into the circuit.
+Returns the previous input [trace].
 
 ## 2.9.10 Z1 Operator
```

Comment:
It's been forever since I've looked at this stuff. Is `Z1` here the same as $z^{-1}$ in the DBSP paper?

Reply:
Yup!